packages/executable/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "executable",
+  "version": "1.0.0",
```

Comment:
We are going to try to keep our package versions in-line for the time being, please make this `0.2.0`